src/xenia/vfs/devices/stfs_xbox.h

```diff
@@ -10,12 +10,38 @@
 #ifndef XENIA_VFS_DEVICES_STFS_XBOX_H_
 #define XENIA_VFS_DEVICES_STFS_XBOX_H_
 
+#include "xenia/xbox.h"
 #include "xenia/base/string_util.h"
 #include "xenia/kernel/util/xex2_info.h"
 
 namespace xe {
 namespace vfs {
 
+// Convert FAT timestamp to 100-nanosecond intervals since January 1, 1601 (UTC)
+inline uint64_t decode_fat_timestamp(const uint32_t date, const uint32_t time) {
+  struct tm tm = {0};
```

Comment:
This file requires `#include <time.h>` now, I think it should be sufficient for `struct tm` and both `_mkgmtime` and `timegm`.

Reply:
Done